nexus/db-model/src/switch_interface.rs

```diff
@@ -118,7 +121,7 @@ pub struct LoopbackAddress {
 
 impl LoopbackAddress {
     pub fn new(
-        id: Option<Uuid>,
+        id: Option<TypedUuid<LoopbackAddressKind>>,
```

Comment:
I feel like `TypedUuid::LoopbackAddress` might be better as an API, even if that causes a lot more noise in the crate.

Reply:
Hmm I'm not quite sure what you mean. `TypedUuid::LoopbackAddress` would be an enum variant (runtime representation), not a type parameter (compile-time representation).